templates/quickstart-database-for-atlassian-services.yaml

```diff
@@ -0,0 +1,179 @@
+---
+# This template is a wrapper for compatible database templates. It exposes a single input/output API and can be referenced the same way no
+# matter which Database "Implementation" is chosen.
+AWSTemplateFormatVersion: 2010-09-09
+Description: RDS Bootstrap template for use with Atlassian Services
+Parameters:
+  DatabaseImplementation:
+    Default: 'AWS Aurora PostgreSQL'
+    Description: 'Database Engine to use. PostgreSQL or AWS Aurora Clustered PostgreSQL'
+    AllowedValues:
+      - 'AWS Aurora PostgreSQL'
+      - 'PostgreSQL'
+    ConstraintDescription: Must be 'true' or 'false'.
+    Type: String
+  DBSecurityGroup:
+    Description: "ID of the security group (e.g. sg-0234se). One will be created for you if left empty."
+    Type: String
+    Default: ''
+  DBAutoMinorVersionUpgrade: 
+    AllowedValues: 
+      - "true"
+      - "false"
+    Default: "false"
+    Description: "Select true/false to setup Auto Minor Version upgrade. e.g. PostgreSQL 9.6.8 -> 9.6.11"
+    Type: String
+  DBBackupRetentionPeriod: 
+    Default: "7"
+    Description: "The number of days for which automatic database snapshots are retained."
+    Type: String
+  DBInstanceClass:
+    AllowedValues:
+      - db.m4.large
+      - db.m4.xlarge
+      - db.m4.2xlarge
+      - db.m4.4xlarge
+      - db.m4.10xlarge
+      - db.m4.16xlarge
+      - db.r4.large
+      - db.r4.xlarge
+      - db.r4.2xlarge
+      - db.r4.4xlarge
+      - db.r4.8xlarge
+      - db.r4.16xlarge
+      - db.t2.medium
+      - db.t2.large
+      - db.t2.xlarge
+      - db.t2.2xlarge
+    ConstraintDescription: Must be a valid RDS instance class from the list
```

Comment:
Should we link to this - https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Concepts.DBInstanceClass.html

Apparently, r4, r5, t3 and t2 can all be used with aurora 🤷‍♂

Reply:
Yeah, however, the dependent template, quickstart-amazon-aurora only accepts r4 instance class at this time.